src/main/java/bizbook/ui/PinnedPersonListPanel.java

```diff
@@ -0,0 +1,66 @@
+package bizbook.ui;
+
+import java.util.logging.Logger;
+
+import bizbook.commons.core.LogsCenter;
+import bizbook.model.person.Person;
+import javafx.beans.property.ObjectProperty;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.Region;
+
+/**
+ * Panel containing the list of pinned persons.
+ */
+public class PinnedPersonListPanel extends UiPart<Region> {
+    private static final String FXML = "PinnedPersonListPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(PinnedPersonListPanel.class);
+
+    @FXML
+    private ListView<Person> pinnedPersonListView;
+
+    /**
+     * Creates a {@code PinnedPersonListPanel} with the given {@code ObservableList}.
+     */
+    public PinnedPersonListPanel(ObservableList<Person> pinnedPersonList, ObjectProperty<Person> focusedPerson) {
+        super(FXML);
+
+        pinnedPersonListView.setItems(pinnedPersonList);
+        pinnedPersonListView.setCellFactory(listView -> new PinnedPersonListViewCell());
+
+        // Add event handler
+        pinnedPersonListView.setOnMouseClicked(event -> handleListViewClick(event, focusedPerson));
+    }
```

Comment:
Just a suggestion or we can do it in a future PR. It would be good to sync the focused person between the two views. So this view should be observing the `focusedPerson` and the `personListPanel` should do the same. Then, when the person is clicked, both sides will highlight the same person.